mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/VeilederflateService.kt

```diff
@@ -107,7 +101,8 @@ class VeilederflateSanityService(
         return when (val result = sanityClient.query(query)) {
             is SanityResponse.Result -> {
                 val gjennomforinger = result.decode<List<VeilederflateTiltaksgjennomforing>>()
-                supplerDataFraDB(gjennomforinger, enhetsId)
+                val gjennomforingerMedDbData = supplerDataFraDB(gjennomforinger, enhetsId)
+                return gjennomforingerMedDbData.filter { filter.lokasjoner.isEmpty() || filter.lokasjoner.contains(it.lokasjon) }
```

Comment:
Filtreringen her er jeg litt usikker på...

Den kan bli hårete når vi evt. flytter andre filter ut fra Groq-spørring og inn i selve logikken for uthenting av gjennomføringer fra db.